include/umf.h

```diff
@@ -15,4 +15,15 @@
 #include <umf/mempolicy.h>
 #include <umf/memspace.h>
 
+// Increment the usage reference counter and initialize the global state of libumf
+// if the usage reference counter was equal to 0.
+// Returns 0 on success or -1 on failure.
+// It must be called just after dlopen() and it is not required in other scenarios.
```

Comment:
shouldn't this comment be like `///` so it becomes visible on the web docs? you could use doxygen syntax as well (e.g. `@brief`)

Reply:
Done